Header header
float32 neckRotat		# left - right rotation
float32 neckPan	        # up - down inclinaison
float32 neckTilt	    # left - right inclinaison
float32 rightLC
float32 leftLC
float32 topLip
float32 bottomLip
float32 rightEyePan
float32 rightEyeTilt
float32 leftEyePan
float32 leftEyeTilt
float32 rightEyeLid
float32 leftEyeLid
float32 rightEar
float32 leftEar


